extensions/amp-bodymovin-animation/validator-amp-bodymovin-animation.protoascii

```diff
@@ -33,6 +33,9 @@ tags: {  # <amp-bodymovin-animation>
     name: "loop"
     value_regex_casei: "(false|number|true)"
   }
+  attrs: {
+    name: "noautoplay"
```

Comment:
may want to have `value: ""` as that would prevent it from just being any value. it'd have to be empty or itself (noautoplay=noautoplay).